.github/workflows/codeql-analysis.yml

```diff
@@ -83,10 +83,11 @@ jobs:
         SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
       run: |
         export MAVEN_OPTS="-Xmx7G -Dorg.slf4j.simpleLogger.defaultLogLevel=warn"
-        sonarProject="-Dsonar.projectKey=idempiere_idempiere -Dsonar.organization=idempiere"
+        sonarProject="-Dsonar.projectKey=motiveth_idempiere -Dsonar.organization=motiveth"
```

Comment:
Hi @hieplq - is this a configuration change for your tests?

Reply:
sorry my mistake
on new commit i use env variable github.repository_owner for it so fork can re-use workflow without modify anything